debian/rules

```diff
@@ -10,19 +10,20 @@ ifeq ($(VERSION_SUFFIX),testing)
 endif
 PACKAGEVERSION := "$(ORIG_VERSION)-$(VERSION_SUFFIX)"
 
-CONFIG := --toolchain=hardened \
-	--prefix=/usr \
+CONFIG := --prefix=/usr \
```

Comment:
I would rather we not pollute `/usr`, stuff all our built files into `/usr/lib/jellyfin-ffmpeg`